cri/v1alpha2/cri.go

```diff
@@ -502,7 +502,7 @@ func (c *CriManager) ListPodSandbox(ctx context.Context, r *runtime.ListPodSandb
 }
 
 // CreateContainer creates a new container in the given PodSandbox.
-func (c *CriManager) CreateContainer(ctx context.Context, r *runtime.CreateContainerRequest) (*runtime.CreateContainerResponse, error) {
+func (c *CriManager) CreateContainer(ctx context.Context, r *runtime.CreateContainerRequest) (_ *runtime.CreateContainerResponse, retErr error) {
```

Comment:
could we use `var err error` to make it readable?

Reply:
Done.